Issue #45 - Do not show unsupported files

armenzg opened the pull request:
r? @jankeromnes

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mozilla/firefox-code-coverage-frontend/180)
<!-- Reviewable:end -->

jankeromnes commented on src/containers/diffViewer.jsx:
```diff
@@ -20,6 +20,21 @@ const sortByPercent = (parsedDiff, coverage) => {
+    if (supportedExtensions.find(ext => ext === extensionTo)) {
+      newDiff.push(p);
+    } else if (supportedExtensions.find(ext => ext === extensionFrom)) {
+      // Case for added files
```
Hm, I don't quite understand, how can added files have a "from" extension? (I'd expect them to just have a "to" and no "from".)

armenzg replied:
I got it the wrong way around. In any case, I've joined both cases.

With regards to tests, we were supposed to have a contributor help with that but she did not do anything. If I have time before the all hands (I will start rewritting AWFY next week)  I want to tackle it since if we want the project to last beyond me being around we should fix it.

jankeromnes commented on src/containers/diffViewer.jsx:
```diff
@@ -20,6 +20,21 @@ const sortByPercent = (parsedDiff, coverage) => {
+    } else if (supportedExtensions.find(ext => ext === extensionFrom)) {
+      // Case for added files
+      newDiff.push(p);
+    }
```
Nit: Maybe log unsupported extensions that we ignore here?

jankeromnes commented on src/containers/diffViewer.jsx:
```diff
@@ -20,6 +20,21 @@ const sortByPercent = (parsedDiff, coverage) => {
+  parsedDiff.forEach((p) => {
+    const extensionTo = p.to.split('.').pop();
+    const extensionFrom = p.from.split('.').pop();
+    if (supportedExtensions.find(ext => ext === extensionTo)) {
```
Suggestion: Maybe you could use `supportedExtensions.indexOf(extensionTo) >= 0` instead of a custom `Array.find()` algorithm?

armenzg commented:
I believe I addressed all the issues and I even added tests.
I've merged the PR since I believe there was no feedback major that seemed to need one more review.
Please correct me if wrong!
